__tests__/ChangePasswordScreen.test.js

```diff
@@ -0,0 +1,114 @@
+import "react-native";
+import React from "react";
+import { mount, shallow } from "enzyme";
+import { act, fireEvent, render, waitFor } from "@testing-library/react-native";
+import ChangePasswordScreen from "../screens/ChangePasswordScreen";
+
+// Note: this is just for use with Jest snapshot testing
+// and comes packaged with react-native init project.
+// You do not need this if using Enzyme 'toMatchSnapshot' etc.
+
+import renderer from "react-test-renderer";
+
+describe("Testing ChangePasswordScreen", () => {
+  it("renders correctly, test using Jest", () => {
+    renderer.create(<ChangePasswordScreen />);
+  });
+
+  it("matches the snapshot", () => {
+    const tree = renderer.create(<ChangePasswordScreen />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("text on change password screen", () => {
+    const wrapper = shallow(<ChangePasswordScreen />);
+    const text = wrapper.find("Text").render().text();
+    expect(text).toEqual("Change your password");
+  });
+
+  it("should say the correct value", () => {
+    const wrapper = mount(<ChangePasswordScreen />);
+
+    const currentPassword = wrapper.find("TextInput").find("#currentPassword");
+    const newPassword = wrapper.find("TextInput").find("#newPassword");
+    const confirmPassword = wrapper.find("TextInput").find("#confirmPassword");
+
+    currentPassword.props().value = "Password";
+    newPassword.props().value = "Peyton12";
+    confirmPassword.props().value = "Peyton12";
+
+    expect(currentPassword.props().value).toEqual("Password");
+    expect(newPassword.props().value).toEqual("Peyton12");
+    expect(confirmPassword.props().value).toEqual("Peyton12");
+  });
```

Comment:
Is this testing the ChangePasswordScreen component, or the TextInput components?

Reply:
This would be testing the input fields in the Form component, thinking it should be its own test?